applications/serial_lte_modem/src/slm_at_host.c

```diff
@@ -213,6 +328,21 @@ static void cmd_send(struct k_work *work)
 		goto done;
 	}
 
+	if (slm_util_cmd_casecmp(at_buf, AT_CMD_SLMUART)) {
+		u32_t baudrate;
+
+		err = handle_at_slmuart(at_buf, &baudrate);
+		if (err != 0) {
+			rsp_send(ERROR_STR, sizeof(ERROR_STR) - 1);
+			goto done;
+		} else {
+			rsp_send(OK_STR, sizeof(OK_STR) - 1);
+			k_sleep(K_MSEC(50));
```

Comment:
Why is the sleep needed before changing baud rate?

Reply:
It is used for sending OK response before changing to new baudrate. Thus the client get OK for the baudrate changing command.